Formula/man-db.rb

```diff
@@ -0,0 +1,45 @@
+class ManDb < Formula
+  desc "Modern, featureful implementation of the Unix man page system"
+  homepage "https://nongnu.org/man-db/"
+  url "https://download.savannah.nongnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  depends_on "libpipeline"
+
+  def install
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-systemdtmpfilesdir=no",
+                          "--with-systemdsystemunitdir=no",
+                          "--disable-cache-owner",
+                          "--disable-setuid"
+    ENV.append_to_cflags "-Wl,-flat_namespace,-undefined,suppress"
```

Comment:
This should be done in configure, not hacked in the formula.

Reply:
Okay, so, what do you want me to do?

Reply:
@Alhadis, could you email me the build failure that results from not including these custom `CFLAGS` (or a link to a failure - I'm not fussy)?  I'm very likely willing to do this upstream, but I want to make sure I properly understand the situation first.

Reply:
Great, thanks.  [Fixed upstream](https://git.savannah.gnu.org/cgit/man-db.git/commit/?id=056e8c7c012b00261133259d6438ff8303a8c36c).  Perhaps you could just leave a comment saying that this can be dropped with >= 2.8.6?

Reply:
Damn that was fast! Thanks! 😀 I've added a note about the removal.